test/functional/sqlsrv/MsSetup.inc

```diff
@@ -44,4 +44,11 @@ if (isset($_ENV['MSSQL_SERVER']) || isset($_ENV['MSSQL_USER']) || isset($_ENV['M
 $keystore = "none";         // key store provider, acceptable values are none, win, ksp, akv
 $dataEncrypted = false;     // whether data is to be encrypted
 
+// for Azure Key Vault
+$keyStoreAuthentication = 'KeyVaultPassword';   // can be KeyVaultPassword or KeyVaultClientSecret
+$principalName = 'name';                        // for use with KeyVaultPassword
+$AKVPassword = 'password';                      // for use with KeyVaultPassword
+$clientID = 'clientid';                         // for use with KeyVaultClientSecret
+$AKVSecret = 'secret';                          // for use with KeyVaultClientSecret
+
```

Comment:
Please be consistent with the other placeholders in this include file (same with the corresponding one for PDO)
So, use a format like `TARGET_***`